### PR TITLE
Add user agent

### PR DESCRIPTION
**Pull request by relango**

### CODE REVIEW

File: src/test/java/com/salesforce/einsteinbot/sdk/examples/SdkExamples.java [hunk 1]
```diff
@@ -1,150 +0,0 @@
-/*
```
Comment (atulkc) [inline src/test/java/com/salesforce/einsteinbot/sdk/examples/SdkExamples.java:1]: I realized that we have `ChatbotClientExamples` and `OAuthExamples` in this package as well. Do we need them? Or can we delete them like this one?

File: src/test/resources/__files/mocks/errorResponse.json [hunk 2]
```diff
@@ -1,8 +0,0 @@
-{
```
Comment (atulkc) [inline src/test/resources/__files/mocks/errorResponse.json:1]: Are these json files left over from some previous test? Don't we need them anymore?

Comment (relango) [reply src/test/resources/__files/mocks/errorResponse.json:1]: Yes, the tests using those files are only part of internal SDK, so getting rid of them in public sdk.

File: src/test/java/com/salesforce/einsteinbot/sdk/examples/SdkExamples.java (hunk 1 above)
Comment (relango) [reply src/test/java/com/salesforce/einsteinbot/sdk/examples/SdkExamples.java:1]: `ChatbotClientExamples` and `OAuthExamples` are useful. I run them after refactoring as well. Maybe we can delete them in the future once we add full working examples in public documentation.

**Review by atulkc — APPROVED** [review]
LGTM

**Pull request merged**